python_modules/dagster/dagster/components/resolved/core_models.py

```diff
@@ -257,6 +258,13 @@ class SharedAssetKwargs(Resolvable):
             ],
         ),
     ] = None
+    freshness_policy: Annotated[
+        Optional[InternalFreshnessPolicy],
+        Resolver.default(
```

Comment:
nit: maybe comment that this is template only, wonder if we should have a shorthand resolver for this  Resolver.template_only or smth